production/helm/loki/templates/networkpolicy.yaml

```
@@ -63,6 +63,9 @@ spec:
         {{- else }}
           - read
           - write
+          {{- if eq (include "loki.deployment.isSingleBinary" .) "true" }}
+          {{- include "loki.singleBinarySelectorLabels" . | nindent 10 }}
```

```suggestion
          single-binary
```

Should be correct, see https://github.com/grafana/loki/blob/afce82071693bb3a90612209decc7afd85227b03/production/helm/loki/templates/single-binary/_helpers-single-binary.tpl#L13